_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1429,6 +1443,37 @@ mfxStatus SetSkipFrame(
     return MFX_ERR_NONE;
 }
 
+mfxStatus VAAPIEncoder::FillPriorityBuffer(mfxPriority& priority)
+{
+    VAStatus vaSts;
+    memset(&m_priorityBuffer, 0, sizeof(VAContextParameterUpdateBuffer));
```

Comment:
Please avoid using memset. Please see https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/Coding_guidelines.md#memory-management

Reply:
hi Dmitry Ermilov. Thanks for your advices. I read the document and modify it after.

Reply:
hi Dmitry Ermilov, memset function is used in other areas. eg: PicParameters. Do you have any better idea？Thanks.

Reply:
Yes, I know. But for new code, better to avoid it.

Reply:
It seems that std::fill is not suitable here.